dsc/src/main.rs

```diff
@@ -1399,6 +1458,7 @@ fn main() -> Result<()> {
             if cleanup {
                 crate::cleanup(output_dir.clone(), region_dir.clone())?;
             }
+
```

Comment:
Nit: bonus whitespace added in this file

Reply:
Removed in 8acd4d6